controller/src/beerocks/bml/internal/bml_internal.h

```diff
@@ -152,6 +152,9 @@ class bml_internal : public beerocks::socket_thread {
     int bml_set_vap_list_credentials(const BML_VAP_INFO *vaps, const uint8_t vaps_num);
     int bml_get_vap_list_credentials(BML_VAP_INFO *vaps, uint8_t &vaps_num);
 
+    //set and get channel scan enable flag
+    int set_dcs_continuous_scan_enable(const sMacAddr &mac, int enable);
+    int get_dcs_continuous_scan_enable(const sMacAddr &mac, int *enable);
```

Comment:
I prefer `int bml_get_dcs_continuous_scan_enable(BML_CTX ctx, const char *radio_mac, int &enable)`

Reply:
the BML cannot use a reference since it's used as a C API
but the bml_internal uses reference